### PR TITLE
Adding langchain client version

### PR DESCRIPTION
**Pull request by dristysrivastava**

### CODE REVIEW

File: pebblo/app/models/models.py [hunk 1]
```diff
@@ -126,6 +126,7 @@ class AiApp(BaseModel):
     lastUsed: datetime
     pebbloServerVersion: Optional[str]
     pebbloClientVersion: Optional[str]
+    clientVersion: Optional[dict]
```
Comment (Raj725) [inline pebblo/app/models/models.py:129]: It seems to be a duplicate of the `framework`(L125), just with a different data type.

Comment (dristysrivastava) [reply pebblo/app/models/models.py:129]: Will use Framework info in clientVersion as 
`clientVersion: Optional[FrameworkInfo]`

And here `langchain_community_version` and its version will get stored

File: tests/app/service/test_discovery.py [hunk 2]
```diff
@@ -50,6 +50,7 @@
     ],
     "framework": {"name": "langchain", "version": "0.1.45"},
     "plugin_version": "0.1.1",
+    "client_version": {"name": "langchain_community_version", "version": "x.x.x"},
```
Comment (srics) [inline tests/app/service/test_discovery.py:53]: nit: `name` should be just pkg name i.e. `langchain_community`
```suggestion
    "client_version": {"name": "langchain_community", "version": "x.x.x"},
```

File: tests/app/service/test_discovery.py [hunk 3]
```diff
@@ -199,6 +200,7 @@ def test_create_ai_apps_model(discovery, mock_pebblo_server_version):
         "lastUsed": datetime.datetime(2024, 1, 1, 0, 0, 5),
         "pebbloServerVersion": "x.x.x",
         "pebbloClientVersion": "0.1.1",
+        "clientVersion": {"name": "langchain_community_version", "version": "x.x.x"},
```
Comment (srics) [inline tests/app/service/test_discovery.py:203]: nit: `name` should be just pkg name i.e. `langchain_community`
```suggestion
        "clientVersion": {"name": "langchain_community", "version": "x.x.x"},
```

**Review by Raj725 — APPROVED** [review]

**Pull request merged**